scale/docs/algorithm_integration/step4.rst

```diff
@@ -1,234 +1,81 @@
-argument.  *If an input is optional, the job will still execute except replace the input with an empty string.* The
-"name" field is a user-defined unique string.  **These input names match the variables in the command_arguments**
-                
-Lines 37-56:  The "output_data" section defines the expected outputs of the job.  The "media_type" defines the expected
-file type of output while the "type" keyword defines whether it expects a file or a list of files.  The "required"
-keyword is a boolean used if the output is guaranteed to be produced by the algorithm on a successful run.  By default,
-the "required" keyword is set to true.  The "name" keyword must be a unique string defining the output.
-**These names are matched with the names in the results manifest to capture results**
-                
-Line 57:      The "shared_resources" keyword is currently unused and reserved for future usage.  It should be set to [].
-
-Line 58:      The "command_arguments" keyword defines what inputs are passed into the docker container.  You can think
-of the arguments as appended to the end of the contents of your dockerfile's ENTRYPOINT for what command is executed.
-The variables here **MUST** match the "name"(s) in your "input_data" section.  The special variable "${job_output_dir}"
-is supplied by Scale and will be your empty output directory Scale will use to search for your results_manifest.json.
-
-Line 59:      The "version" is an optional string defining the version of the definition specification used.  This
-defaults to the latest version.
-
-Line 60:      The "command" field defines the main command to execute on the command line.
-*This should match your ENTRYPOINT in your dockerfile*.
+Adding your new job into a Scale job type
+=========================================
+
+After you have successfully tested your Docker container and inserted the job into the database, you can add the job to an existing recipe or create a brand new recipe. Recipes allow you to chain the inputs/outputs of one job to another job to create a workflow for your algorithm processing.
+
+
+Creating a new job type
+-----------------------
+
+New job types are added from the Scale web interface in the Configuration->Job Types section. *You will need to be logged in as an administrator in order to add new recipes.*
+
+Selecting the “Create Job Type” button will display an empty job type template.
+
+1. **Seed Image** is the Seed image that this job will process.
+
+2. **General Information** is where you select an icon to be used for the job type in Scale.
+
+3. **Validate and Create** is the recipe version number. You should use semantic versioning, e.g. 1.2.3.
+
+4. **Description** is a brief summary of the recipe and what it should create.
+
+
+Adding a trigger
+----------------
+
+After adding a name, title, version, and description of your recipe, you can add the recipe trigger that will be used to run your recipe.
```

Comment:
Remove triggers? Or is this for 5.9.x?

Reply:
The concept of triggers is still around and Strikes/Scans still trigger a recipe to be ran. They are just linked directly to the recipe to be ran instead of matching to a job/recipe by data/file type.